src/code/z_fbdemo_dlftbls.c

```diff
@@ -1,5 +1,53 @@
 #include "global.h"
+#include "overlays/fbdemos/ovl_fbdemo_triforce/z_fbdemo_triforce.h"
+#include "overlays/fbdemos/ovl_fbdemo_wipe1/z_fbdemo_wipe1.h"
+#include "overlays/fbdemos/ovl_fbdemo_wipe3/z_fbdemo_wipe3.h"
+#include "overlays/fbdemos/ovl_fbdemo_wipe4/z_fbdemo_wipe4.h"
+#include "overlays/fbdemos/ovl_fbdemo_wipe5/z_fbdemo_wipe5.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_fbdemo_dlftbls/func_80163C90.s")
+#define TRANSITION_OVERLAY(name, filename)                                                                       \
+    {                                                                                                            \
+        { 0, 0 }, SEGMENT_START(ovl_##filename), SEGMENT_END(ovl_##filename), SEGMENT_ROM_START(ovl_##filename), \
+            SEGMENT_ROM_END(ovl_##filename), &name##_InitVars, sizeof(name)                                      \
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_fbdemo_dlftbls/func_80163D80.s")
+#define TRANSITION_OVERLAY_INTERNAL(name) \
+    { { 0, 0 }, NULL, NULL, 0, 0, &name##_InitVars, sizeof(name) }
+
+#define TRANSITION_OVERLAY_UNSET \
+    { 0 }
+
+TransitionOverlay gTransitionOverlayTable[] = {
+    TRANSITION_OVERLAY_INTERNAL(TransitionFade),       TRANSITION_OVERLAY(TransitionTriforce, fbdemo_triforce),
+    TRANSITION_OVERLAY(TransitionWipe1, fbdemo_wipe1), TRANSITION_OVERLAY(TransitionWipe3, fbdemo_wipe3),
```

Comment:
I know it got discussed in the discord a bit, but personally I think it would be best to just capitalize the directories/whatever else is needed and simplify the macro. Though should not hold up the PR on this point.

Reply:
I've changed my mind on this one. I think it's better to have the file name in there separately from the name of the struct, even if the directories were properly capitalized.

Reply:
I think it would be better to have the original filename in the macro so it will be easier for us to rename them in the future